spec/src/utils/helpers.js

```diff
@@ -45,6 +47,79 @@ describe('ConstructorIO - Utils - Helpers', () => {
       });
     });
 
+    describe('toSnakeCase', () => {
+      it('Should convert to camel case', () => {
+        const camelCasedStr = 'helloThisIsMyWorld';
+        const snakeCasedStr = toSnakeCase(camelCasedStr);
+
+        expect(snakeCasedStr).to.equal('hello_this_is_my_world');
+      });
+
+      it('Should not modify snake case', () => {
+        const camelCasedStr = 'a_lovely_day';
+        const snakeCasedStr = toSnakeCase(camelCasedStr);
+
+        expect(snakeCasedStr).to.equal('a_lovely_day');
+      });
+    });
+
+    describe('toSnakeCaseKeys', () => {
+      it('Should convert shallow keys', () => {
+        const camelCasedObj = {
+          originReferrer: 'https://test.com/search/pizza?a=bread&b=pizza burrito',
+          userId: 'boink doink yoink',
+          section: 'Products',
```

Comment:
Should we add an additional key here that maps to an object with camelCased key value pairs? Just wanted to make sure we're testing the case where not passing `true` as the 2nd parameter to `toSnakeCaseKeys` works as expected.